core/src/scene/sceneLoader.cpp

```diff
@@ -1249,6 +1253,48 @@ StyleUniforms SceneLoader::parseStyleUniforms(const Node& value, Scene& scene) {
     return std::make_pair(type, std::move(uniformValues));
 }
 
+void SceneLoader::parseTransition(Node params, Scene& scene, std::vector<StyleParam>& out) {
+
+    static const std::string prefix = "transition";
+
+    for (const auto& prop : params) {
+        switch (prop.first.Type()) {
+            case YAML::NodeType::Sequence:
+            {
+                auto rawKeys = parseSequence(prop.first);
+                std::stringstream ss(rawKeys);
+                std::vector<std::string> keys;
+                std::string key;
+
+                while (std::getline(ss, key, ',')) {
+                    keys.push_back(key);
+                }
```

Comment:
More briefly:
```c++
auto keys = prop.first.as<std::vector<std::string>>();
```
We could also pull `keys` and the loop over `keys` outside the switch/case and just populate`keys` inside the cases, if that makes sense to you. 

Just trying to keep `SceneLoader` on a diet :)